### ISSUE TITLE
"winget source export" exports invalid json

### ISSUE THREAD

**Issue by palenshus** [microsoft/winget-cli#1688]
### Brief description of your issue

The `winget source export` command outputs invalid json, specifically what appears to be an array of json objects, but without having the correct [] wrapping or commas between the objects.

### Steps to reproduce

Run `winget source export` with more than one source configured	

### Expected behavior

Should output a valid json array, like below. Pretty-printed for readability, not necessary in final impl though:

```json
[
    {
        "Arg": "https://localhost:7071/api/",
        "Data": "",
        "Identifier": "winget-cli-restsource-dev",
        "Name": "winget-pkgs-restsource-local",
        "Type": "Microsoft.Rest"
    },
    {
        "Arg": "https://winget.azureedge.net/cache",
        "Data": "Microsoft.Winget.Source_8wekyb3d8bbwe",
        "Identifier": "Microsoft.Winget.Source_8wekyb3d8bbwe",
        "Name": "winget",
        "Type": "Microsoft.PreIndexed.Package"
    }
]

```

### Actual behavior

In my case it dumps this output:

```json
{"Arg":"https://localhost:7071/api/","Data":"","Identifier":"winget-cli-restsource-dev","Name":"winget-pkgs-restsource-local","Type":"Microsoft.Rest"}
{"Arg":"https://winget.azureedge.net/cache","Data":"Microsoft.Winget.Source_8wekyb3d8bbwe","Identifier":"Microsoft.Winget.Source_8wekyb3d8bbwe","Name":"winget","Type":"Microsoft.PreIndexed.Package"}
```

### Environment

```shell
Windows Package Manager v1.1.12986
Copyright (c) Microsoft Corporation. All rights reserved.

Windows: Windows.Desktop v10.0.22000.258
Package: Microsoft.DesktopAppInstaller v1.16.12986.0
```

**Comment by jedieaston** [comment]
I think this may have been done so that they could be copy pasted into the GPO, since that policy takes in an array of JSON objects:

![image](https://user-images.githubusercontent.com/21368066/140958529-e58ba173-fc25-4051-8580-0c11c2a34230.png)

That policy should probably be adjusted to just take in a JSON array, but that will break the current policy (since it's in preview I don't know if that's subject to some kind of deprecation thing).

**Comment by denelon** [comment]
@jedieaston, you are correct. The output was designed to make policy easy. If it's not valid JSON, we need to take a look at a better mechanism. This would be a breaking change.

**Comment by lanoglide** [comment]
I don't have too much to say but around some alter ego stereo-typical outlandish bondage ... is reputable for according to an ol' insufficient list to command to the core of such rivalry, I commend such prudence as a light to a soon becoming a beautiful thing... Anywho, if I would, but only pay my respects I shall amend you this. "majorgeeks.com; patchmypc.com; softwareinformer.com;  sourceforge.net; androidfilehost.com; download.cnet.com;" #Tah, tah

**Comment by Masamune3210** [comment]
.....what?

**Comment by lechacon** [comment]
As @jedieaston and @denelon said, this was intended to be used when managing group policies. I believe in that scenario it would be more common to work with individual sources than with the full set of sources. For example, if I have sources A, B, C and D, I may want to add A to the list of allowed sources, add B to the list of required sources, do nothing with C because I don't want it, and do nothing with D because it is already in the group policy. So it would be easier if the output was multiple independent JSON objects that I can manage independently of one another (current behaviour), than having a JSON array and having to edit the square braces and commas.

For the same reason it is all in a single line and not pretty-printed. As far as I remember the group policy editor wasn't great for working with data having multiple lines, if it was at all possible. (And since it is backed in a registry value, I don't think it was possible at all.)

Having a single array that is pretty-printed may be better for other uses, but I don't think this is currently used for anything else?

**Comment by palenshus** [comment]
I was hoping to use the command in an automated test to dump the list of currently added sources, deserialize and inspect the result to determine if my test source was already added, and if not, add it. It's possible currently, just requires some string manipulation or raw text search. I can live without this
